frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -125,7 +168,9 @@ const ResolutionResultByPostorderID& resolveModule(Context* context, ID id) {
       auto modScope = scopeForId(context, mod->id());
       emitMultipleDefinedSymbolErrors(context, modScope);
 
-      result.setupForSymbol(mod);
```

Comment:
Did you intend to delete the code that sets `symbol` in the `result`? If so, why?

Reply:
Yes because it is done in the new `Resolver::createForModuleStmt` added immediately after. I'll put it back if you think it's clearer -- I don't think there's any harm in setting it twice.

Reply:
Learning that it's done in `createForModuleStmt` makes me feel fine about it. I just saw the red and got worried :-)